Src/ILGPU/Runtime/Cuda/CudaNativeMethods.cs

```diff
@@ -284,7 +285,69 @@ public enum MemHostRegisterFlags
         CU_MEMHOSTREGISTER_READ_ONLY = 8,
     }
 
+    /// <summary>
+    /// Controls the behaviour of <see cref="CudaAPI.cuIpcOpenMemHandle">IpcOpenMemHandle</see>
+    /// </summary>
+    [Flags]
+    public enum CudaIpcMemFlags
+    {
+        /// <summary>
+        /// No flags.
+        /// </summary>
+        None,
+        /// <summary>
+        /// Enables peer access lazily.
+        /// </summary>
+        /// <remarks>
+        /// From <a href="https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__TYPES.html#group__CUDART__TYPES_1g60f28a5142ee7ae0336dfa83fd54e006">
```

Comment:
ISSUE: I'm guessing the GUID will change with each Cuda release, so I would prefer not to include the link.

Reply:
Understandable, I also removed all of the remarks as well, since they seemed too vague without the links.